src/providers/clipboard/clipboard.ts

```diff
@@ -46,19 +47,21 @@ export class ClipboardProvider {
       return this.clipboard.paste();
     } else if (this.isNW) {
       return this.nodeWebkitProvider.readFromClipboard();
+    } else if (this.isDevel) {
+      return (navigator as any).clipboard.readText();
```

Comment:
It doesn't work on any browser: https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/readText